### PR TITLE
🐛 Be able to accept notStats option in form list

### PR DESCRIPTION
**Pull request by grdryn**
This option has been recognized for a while -- form submission storage
use won't be calculated if it's set, but it wasn't clear how to set
it. This change accepts it as a query param to the list middleware
function.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,7 +25,7 @@
     "fh-cls-mongoose": "2.1.1",
     "fh-gridfs": "1.1.1",
     "fh-logger": "0.5.0",
-    "fh-mbaas-client": "0.15.0",
+    "fh-mbaas-client": "github:grdryn/fh-mbaas-client#process-notStats",
```
Comment (grdryn) [inline package.json]: I'll update this when https://github.com/feedhenry/fh-mbaas-client/pull/29 is merged and published

**Comment by grdryn** [comment]
I haven't done the "don't bother calculating storage if there are more than 10,000 submissions" part here yet

File: package.json [hunk 2]
```diff
@@ -25,7 +24,7 @@
     "fh-cls-mongoose": "2.1.1",
     "fh-gridfs": "1.1.1",
     "fh-logger": "0.5.0",
-    "fh-mbaas-client": "0.15.0",
+    "fh-mbaas-client": "1.1.1",
```
Comment (davidffrench) [inline package.json]: Is this bump in fh-mbaas-client needed? I thought it was fh-supercore > fh-mbaas-client > fh-mbaas > fh-forms.

Comment (grdryn) [reply package.json]: Needed? Probably not. I don't see why we wouldn't bump it though, given that I've made changes to fh-mbaas-client for fh-supercore's interaction with it, as part of this same body of work. If we _don't_ bump it, then it just falls further behind, and results in 2 different copies of fh-mbaas-client (& probably some dependencies) below fh-supercore. When they're the same version, they'll be deduplicated.

**Comment by grdryn** [comment]
Now that `fh-mbaas-client@1.1.1` has been published, I retriggered the builds here, but they failed because of eslint and a test. I've fixed both (added es6 to eslint, and test stub just needed to be promisified). The builds have passed now, so I'll merge and publish, and update the downstream PRs accordingly.

**Pull request merged**